field_points/Kenya/README.md

```diff
@@ -14,28 +14,57 @@
 <a title="TUBS, CC BY-SA 3.0 &lt;https://creativecommons.org/licenses/by-sa/3.0&gt;, via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File:Kenya,_administrative_divisions_-_de_-_colored.svg"><img width="300" alt="Kenya, administrative divisions - de - colored" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/54/Kenya%2C_administrative_divisions_-_de_-_colored.svg/512px-Kenya%2C_administrative_divisions_-_de_-_colored.svg.png"></a>
 
 ### Geography
+**Location** <sup>[1]</sup>
+- Eastern Africa
+- Bordering the Indian Ocean
+- Between Somalia & Tanzania
+
 **Area** <sup>[1]</sup>
 - Total: 580,367 sq km
 - Land: 569,140 sq km
 - Water: 11,227 sq km
 
-**Landscape**
-
+**Landscape** <sup>[3]</sup>
+1. Lake Victoria Basin: part of a plateau rising eastward from the lakeshore to the Rift highlands
+2. Great Rift Valley & Associated Highlands: splits the highland region into two sections: Mau Escarpment to the west & Aberdare Range to the east, valley itself is 30 to 80 miles (50 to 130 km) wide, important in historic & economic development, region was focus of European settlement
+3. Eastern Plateau Forelands: just east of Rift highlands, constitute a vast plateau of ancient rocks gently sloping to the coastal plain, scattered hills & striking elevated formations
+4. North & Northeast Semi-arid/Arid Regions: vast region extending from Ugandan border through Lake Rudolf to the plateau area between the Ethiopian and Kenyan highlands, movement of people & livestock is limited by availability of water
+5. Coastal Region: runs for about 250 miles (400 km) along the Indian Ocean, mangrove swamps, low-lying coastal areas
 
-**Terrestrial Ecoregions** 
-<!-- 1. Guinean forest-savanna mosaic
+**Terrestrial Ecoregions**
+1. Guinean forest-savanna mosaic
 2. Sahelian Acacia savanna
-3. West Sudanian savanna, and
-4. Guinean mangroves -->
+3. West Sudanian savanna
+4. Guinean mangroves
+
+**Land Use** <sup>[1]</sup>
+- Agricultural land: 48.1% (2018 est.)
+- Arable land: 9.8% (2018 est.)
+- Permanent crops: 0.9% (2018 est.)
+- Permanent pasture: 37.4% (2018 est.)
+- Forest: 6.1% (2018 est.)
+- Other: 45.8% (2018 est.)
 
-**Climate** 
-<!-- - Rainy season: May to November
-- Dry season: December to April -->
+**Climate**
+- Varies from tropical along coast to arid in interior
+- Seasonal climatic changes controlled by large-scale pressure systems of western Indian Ocean and adjacent landmasses
+- Long rain season: March to May
+- Short rain season: October to November
 
-**Soils** 
-<!-- - In northwest soils are highly favorable for peanut cultivation
+**Natural Resources** <sup>[1]</sup>
+
+Limestone, Soda ash, Salt, Gemstones, Fluorspar, Zinc, Diatomite, Gypsum, Wildlife, Hydropower
```

Comment:
I am curious about your reasoning for adding natural resources. There probably is some connection to agriculture but I am unaware of it currently.

Reply:
I think I added it b/c it fell under the geography category in the cia source but I realized it doesn't have much of a connection to agriculture so I took it out!

Reply:
Okay, I think it hints a little at the soil and land cover but agree that the connection is a distant one.